app/models/user.rb

```diff
@@ -32,11 +32,11 @@ def video_queued?(video)
   end
 
   def follow(other_user)
```

Comment:
I would probably make this a bang method since it affects change. Also, it is a good idea to use the bang version of `save` and `create` (line 35) when you are not checking if the creation was successful. Currently line 35 will fail gracefully and you won't get an alert about it. You want it to raise an error so you know about it.